activerecord/lib/active_record/model_schema.rb

```diff
@@ -269,6 +298,18 @@ def inheritance_column
         (@inheritance_column ||= nil) || superclass.inheritance_column
       end
 
+      def define_attributes_from_schema
+        if defined?(@define_attributes_from_schema)
+          @define_attributes_from_schema
+        else
+          superclass.define_attributes_from_schema
+        end
+      end
+
+      def define_attributes_from_schema=(value)
+        @define_attributes_from_schema = value
+      end
```

Comment:
Isn't this mimicking the behaviour of `class_attribute`?  Could we instead use `class_attribute : define_attributes_from_schema, instance_accessor: false, default: true`?